src/main/java/Assignment9.java

```diff
@@ -0,0 +1,11 @@
+public class Assignment9 {
+    public static void main(String[] args) {
```

Comment:
Instead of writing this code in void main, lets define a function for this. 
and lets create test in src/test/* package which tests your function